how-to/common/public/components/fin-toggle-class.js

```diff
@@ -0,0 +1,124 @@
+customElements.define(
+  "fin-toggle-class",
+  class ToggleClass extends HTMLElement {
```

Comment:
I think it needs a different name, something like `ThemeToggle` , `ToggleClass` is a bit too generic

Reply:
Also would be better derived from HTMLButtonElement to be more behaviourally correct (might need some styling removed to look plain)

Reply:
ToggleClass is intentionally generic. Theme toggle is what we are using it for but you have the option of using it to toggle two classes against any element and specify an image that represents that. ThemeToggle might make people think it applies to the platform as a whole (from a shared component perspective) and it would involve more than putting a class on the body element.

Reply:
Updated to a button with a style.